modules/integration/tests-ui/src/test/resources/automation.xml

```diff
@@ -0,0 +1,190 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+  ~  Copyright (c) 2005-2010, WSO2 Inc. (http://wso2.com) All Rights Reserved.
```

Comment:
Fixed in https://github.com/wso2/product-app-manager/commit/f021f86c3c398dddeeca17e225a37e42f7f7033b